cmd/http-server/startcmd/start.go

```diff
@@ -317,10 +327,25 @@ func fetchTrustBlocWASMAgentOpts(cmd *cobra.Command) (*trustblocAgentJSOpts, err
 		return nil, err
 	}
 
+	// This is used for storage of docs in SDS.
+	// TODO get username from the actual registration process instead of a cmd line arg #266
+	agentUsername, err := cmdutils.GetUserSetVarFromString(
+		cmd, agentDefaultLabelFlagName, agentDefaultLabelEnvKey, false)
+	if err != nil {
+		return nil, err
+	}
+
+	sdsServerURL, err := cmdutils.GetUserSetVarFromString(cmd, sdsURLFlagName, sdsURLEnvKey, false)
```

Comment:
is field mandatory?

Reply:
Yes - SDS is our permanent storage. Without it, all the user's stuff would be lost as soon as the cache clears. It's part of the overall larger picture with hub-auth integration, logging in, syncing wallet data, etc